### ISSUE TITLE
Specific Date Selector

### ISSUE THREAD

**Issue by JWolfcat** [liamcain/obsidian-calendar-plugin#294]
Hi!
As the title says I miss a way to select exact dates to make notes on.

## Current Workaround
I have been using this workaround: 
I have a note with the date as title (Like it was daily note template), then I use the "reveal active note" command and it send me to the correct date and finally I create the new note in the date and copy-paste the content, if it has some. 
By this method I have to create the note first and then create it again as a daily note, and I would love to search for the date first and then create the daily note directly.
***
# Possible Solutions
I have, two possibilities in mind, the "fancy solution", and the "command solution"

## The fancy solution
Imagine you want to transcribe an old diary from Oct 10, 99', now to create the note into calendar you have to go 23 years back times 12 moth per year using the arrows (Or use the workaround).

I would love to have a date selection system like Windows 11 calendar widget, it would be like this:
Clicking on the moth/year at the top left, the calendar "zooms out" into yearly view, with all months. Then if you click on the year (top left too) it would "zoom out" again into a year list where you can scroll to go to the selected year.
Going to Oct 10, 99' would be only 4 or 5 clicks to find: 

From "today" to 1999/10/10
- 2 click to see year list
- scroll and find 1999
- click to enter it
- click on October
- click the 10th to create the new note
This is just a 

## The command solution
Have a command like "Go to date" or "Create a note on specific date" witch ask you to fill "day", "month" and "year". That would do the work.
I think this is the easiest way to implement this, but I have no idea of coding, so its not my business to judge this.
***
I've being looking for another way to achieve this and I haven't find it (Please, let me know if there is another way already ) , but having this feature would be a life changer.

Thanks for your work.

**Comment by esamecar** [comment]
Yeah, it's not that easy to go back to e.g. 2010 right now ... Would love to see an option that makes this easier.

**Comment by BabakFarshchian** [comment]
I add my vote!

**Comment by Elaws** [comment]
This is absolutely needed. It's so unfortunate that the plugin doesn't seem to be maintained anymore !
